bin/drone/destroy-errored-agents

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+
+# This script will terminate all drone agents that are in 'error' state. This
+# happens occasionally, and we have not been able to pin down the exact reason
+# why. This was created when we were using the following versions, and may not
+# be needed if future bugfixes resolve this issue.
+#
+# - drone/autoscaler:1.8.2
+# - drone/drone:1.9.0
+#
+# If it's too fancy for you, you can always run the following command.
+#
+# ```
+# drone server ls -l | grep error | awk '{print $1}' | xargs -tL1 drone server destroy`
```

Comment:
Might be worth adding a link to https://docs.google.com/document/d/1Qls20xfNN6T_DErOMwVQOFJsxAEAdMWFHzBDIbxyUQQ/edit#heading=h.9bln2fv2lgqd, in an effort to prevent these two sources of information from becoming out of sync

Reply:
I'm hesitant to link to a private google doc from our repo, as I don't want us to get share requests from random people.

We got this a lot for the AWS Access doc, so I think I'll do the same thing here which is to make sure the doc is clearly organized and named in our google drive, then say something like "See the 'Drone Continuous Integration' doc in our Engineering drive."

Reply:
Good point! I think that'll satisfy the need